framework/tests/viewpointtest/VarargsConstructor.java

```diff
@@ -2,15 +2,14 @@
 // https://github.com/eisop/checker-framework/issues/777
 import viewpointtest.quals.*;
 
-public class VarargsConstructor {
-
+@ReceiverDependentQual public class VarargsConstructor {
```

Comment:
Should there be two variants of the test? Why is it enough to add RDQ here? Some comments would be helpful to understand these tests.

Reply:
I think right now viewpoint test checker has a lot suppress warnings and errors because of we default everything to Top. Since we also forbid Top object creation, I would like to make viewpoint test checker more PICO like system, i.e. default Object to RDQ and make A as the default qualifier.